main.pm

```diff
@@ -221,6 +221,10 @@ sub system_is_livesystem() {
   return (check_var("FLAVOR", 'Rescue-CD') || get_var("LIVETEST"))
 }
 
+sub is_server() {
+  return (get_var("OFW") || get_var("FLAVOR", '') eq "Server-DVD");
```

Comment:
just curious, why not use check_var("FLAVOR", "Server-DVD") here? check_var() is exact designed for this kind case